NoJones/Controller/InitialScreenViewController.swift

```diff
@@ -33,8 +31,8 @@ class InitialScreenViewController: UIViewController{
         }
         else {
             performSegue(withIdentifier: "segue", sender: nil)
-            userData.set(nameTextField.text, forKey: "userName")
-            userData.set(bornDate.date, forKey: "bornDate")
+            guard let name = nameTextField.text, let bornDate = bornDate?.date else { return }
+            UserDefaultsManager.setUser(name: name, bornDate: bornDate)
```

Comment:
Já que toda vez você passa um optional o próprio método do userDefault poderia receber um ```?```  e tratar lá. Evita ficar duplicando o ``` guard let ``` toda vez que for usar o método.

Reply:
essa foi uma boa

Reply:
n tinha pensado nisso